src/main/java/com/rtb/projectmanagementtool/user/UserProfileServlet.java

```diff
@@ -109,14 +109,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
       throws ServletException, IOException {
 
     // get parameters
-    long taskID = Long.parseLong(request.getParameter("taskID"));
```

Comment:
this is null even though I thought I was passing it from user-private-comments.jsp. the task (which I printed out in the jsp) is not null, so this is just not being passed.